src/views/partials/header.hbs

```diff
@@ -3,6 +3,5 @@
       <meta name="viewport" content="width=device-width,
       initial-scale=1.0">
       <meta http-equiv="X-UA-Compatible" content="ie=edge">
-      <link rel="stylesheet" href="/css/style.css">
```

Comment:
???

Reply:
What's your comment here @Karyum?
How shall we act on this?

Reply:
i was just wondering what happend here why was this delete in this pull request ?